### PR TITLE
URL Validation

### PR DESCRIPTION
**Pull request by FritzHoing**
Implement an URL validation.
Whenever an invalid URL is used, a short information will be displayed.

### CODE REVIEW

File: src/i18n/translations.ts [hunk 1]
```diff
@@ -252,7 +257,10 @@ export default {
         columnTitle: 'Name',
         errorMessage: 'Error',
         errorDescription: 'Could not load the provided Capabilities document, please check the validity of the URL',
-        version: 'Version'
+        invalidUrlErrorMsg: 'Invalid URL specified. Please provide an URL like https://ows.terrestris.de/osm/service',
+        version: 'Version',
+        requestWmsGetCapabilitiesInstruction: 'Please provide any valid GetCapabilities URL und submit ' +
```
Comment (marcjansen) [inline src/i18n/translations.ts:262]:
```suggestion
        requestWmsGetCapabilitiesInstruction: 'Please provide a valid GetCapabilities URL and submit ' +
```

File: src/i18n/translations.ts [hunk 2]
```diff
@@ -252,7 +257,10 @@ export default {
...
+          'your input with \'Enter\' or click on search button.'
```
Comment (marcjansen) [inline src/i18n/translations.ts:263]:
```suggestion
          'your input by pressing the \'Enter ↲\' key or by clicking on the button right of the field.'
```

File: src/i18n/translations.ts [hunk 3]
```diff
@@ -15,7 +15,12 @@ export default {
         columnTitle: 'Name',
         errorMessage: 'Fehler',
         errorDescription: 'Das Capabilities Dokument konnte nicht geladen werden, bitte prüfen Sie die URL',
-        version: 'Version'
+        invalidUrlErrorMsg: 'Ungültige URL. Bitte geben Sie eine gültige URL wie z.B. ' +
+          'https://ows.terrestris.de/osm/service ein.',
+        version: 'Version',
+        requestWmsGetCapabilitiesInstruction: 'Bitte geben Sie eine gültige WMS GetCapabilities Adresse ein ' +
+          'und bestätigen Sie die Eingabe mit \'Enter\' oder über den Button rechts vom Eingabefeld.'
```
Comment (marcjansen) [inline src/i18n/translations.ts:22]:
```suggestion
          'und bestätigen Sie die Eingabe mit \'Enter ↲\' oder über den Button rechts vom Eingabefeld.'
```

File: src/components/AddLayerModal/index.tsx [hunk 4]
```diff
@@ -158,6 +172,8 @@ export const AddLayerModal: React.FC<AddLayerModalProps> = ({
     closeModal();
   };
 
+  console.log(validationStatus)
```
Comment (marcjansen) [inline src/components/AddLayerModal/index.tsx:175]:
```suggestion
```

File: src/components/AddLayerModal/index.tsx [hunk 5]
```diff
@@ -41,6 +46,8 @@ import {
 } from '../../store/addLayerModal';
 
 import './index.less';
+import { SimpleButton } from '@terrestris/react-geo';
+import { log } from 'console';
```
Comment (marcjansen) [inline src/components/AddLayerModal/index.tsx:50]:
```suggestion
```

File: src/components/AddLayerModal/index.tsx [hunk 6]
```diff
@@ -41,6 +46,8 @@ import {
 } from '../../store/addLayerModal';
 
 import './index.less';
```
Comment (marcjansen) [inline src/components/AddLayerModal/index.tsx:48]: Please organize and probably also sort imports.

File: src/components/AddLayerModal/index.less [hunk 7]
```diff
@@ -8,4 +22,12 @@
     padding-top: 0.5rem;
   }
 
+  .ant-typography.error {
+    color: red;
+  }
+
+  .url-input {
+    padding: 10px 0;
```
Comment (marcjansen) [inline src/components/AddLayerModal/index.less:30]:
```suggestion
    padding: 1em 0;
```
untested, but relative units (like `em`, `rem` etc.) should be preferred.

**Review by marcjansen — APPROVED** [review]
Only nit-pick, please take what you agree with.

The linked PR is not accessible to most people. please remove it from the PR-description.

File: src/i18n/translations.ts (hunk 2 above)
Comment (FritzHoing) [reply src/i18n/translations.ts]: Thank you very much! I went along with your suggestion.

File: src/components/AddLayerModal/index.tsx (hunk 4 above)
Comment (FritzHoing) [reply src/components/AddLayerModal/index.tsx]: Sorry, that somehow sneaked it way into the MR.

File: src/components/AddLayerModal/index.less (hunk 7 above)
Comment (FritzHoing) [reply src/components/AddLayerModal/index.less]: Thank you! This is now tested and approved

File: src/components/AddLayerModal/index.tsx (hunk 6 above)
Comment (FritzHoing) [reply src/components/AddLayerModal/index.tsx]: The imports are now organized!

**Review by marcjansen — APPROVED** [review]
In with this. Thanks @FritzHoing

**Pull request merged**